bin/mail/2016-11-10-moving-forward-together/generate-teachers

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+
+# All US teachers (based on the teacher queries from 2016-11-03-hoc-global-site-refresh)
+query = '
+  (
+    location_country_s: "united states" ||
+    location_country_code_s: us ||
+    hoc_country_s: us ||
+    country_s:"united states" ||
+    create_ip_country_s: "united states" ||
+    create_ip_country_s: reserved
+  ) &&
+  (
+    kind_s: HocSignup2016 ||
+    kind_s: HocHardwarePrizes2014 ||
+    kind_s: HocHardwarePrizes2015 ||
+    kind_s: HocSignup2014 ||
+    kind_s: HocSignup2015 ||
+    kind_s: HocSurvey2014 ||
+    kind_s: HocSurvey2015 ||
+    kind_s: CSEdWeekEvent2013 ||
+    (
+      kind_s: user && role_s: teacher
+    ) || (
+      kind_s: Petition && role_s: educator
+    ) ||
+    kind_s: BringToSchool2013 ||
+    kind_s: ClassSubmission ||
+    kind_s: DistrictPartnerSubmission ||
+    kind_s: PLP interest form ||
+    kind_s: Teacher interest form ||
+    kind_s: School interest form ||
```

Comment:
Hmm. Actually those aren't form types (space or no space). I don't know what these are :( We didn't catch that earlier [here](https://github.com/code-dot-org/code-dot-org/blob/a7ae172ebb006f34241f75c784df4bf54364a84e/bin/mail/2016-11-03-hoc-global-site-refresh/generate-teachers#L54-L56). I'll just remove them from this query.